lib/config.ts

```diff
@@ -1,3 +1,5 @@
+export interface HttpProxyAgent { constructor(opts: Object): HttpProxyAgent; }
```

Comment:
Why not just type that as `any`? I'm not sure why this particular one requires a special type.